src/components/input-dropdown/input-dropdown.component.spec.ts

```diff
@@ -237,6 +239,35 @@ describe('InputDropdownComponent', () => {
         expect(openChangeSpy).toHaveBeenCalledWith(false);
     });
 
+    it('should not open the dropdown when disabled is true', async () => {
+        component.disabled = true;
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        const dropdown = document.querySelector('.filter-container');
+        expect(dropdown).toBeNull();
+    });
+
+    it('should not focus the input field when disabled is true.', async () => {
```

Comment:
I don't think there is any point to this test because the input field doesn't even exist when the dropdown isn't open. What you should test is that the control itself (the button) cannot receive focus when disabled.